sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamComplexTypeTest.java

```diff
@@ -373,6 +376,45 @@ public void testNullInnerRow() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(2));
   }
 
+  @Test
+  public void testLogicalTypes() {
+    Schema beamSchemaWithLogicalTypes =
+        BigQueryUtils.fromTableSchema(
```

Comment:
Is it possible to use `DummySqlTimeType`  and `DummySqlDateType` as in other tests? I find dependency on gcp-io not necessary